src/plugins/core/merge.ts

```diff
@@ -367,6 +369,13 @@ export class MergePlugin extends CorePlugin<MergeState> implements MergeState {
     return CommandResult.Success;
   }
 
+  private checkTargetValidity({ target }: AddMergeCommand | RemoveMergeCommand): CommandResult {
+    if (target.length === 0) {
```

Comment:
Do we have any commands that would be valid with an empty target ? If no, this could be bumped in the `allowDispatch` of the sheet plugin

Reply:
I don't think so, I haven't really thought it through but imagine a chart-like command with a target and you explicitely empty the list of range for whatever reason. I think it could vary from one use to the other

Reply:
> https://www.odoo.com/odoo/2328/tasks/4863945